packages/monitoring-system/data-processor/cloudbuild.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/npm
```

Comment:
You can probably skip this if the compile already happens in the docker build

Reply:
I don't believe it does: https://github.com/azizsonawalla/repo-automation-bots/blob/automate-deployments-2/packages/monitoring-system/data-processor/Dockerfile

The Docker container copies the compiled output and directly runs `npm start`

Reply:
I see, is this to separate the dev dependencies from the production/runtime ones and limit the size of the image?

Reply:
No, there was no particular reason other than that's how the Cloud Run docs did it